bot/exts/holidays/valentines/lovecalculator.py

```diff
@@ -90,7 +90,7 @@ def normalize(arg: Member) -> Coroutine:
             name="A letter from Dr. Love:",
             value=data["text"]
         )
-        embed.set_footer(text=f"You can unsubscribe from lovefest by using {Client.prefix}lovefest unsub")
+        embed.set_footer(text="You can unsubscribe from lovefest by using !subscribe")
```

Comment:
```suggestion
        embed.set_footer(text="You can unsubscribe from lovefest by using `!subscribe`.")
```

Not necessary, but maybe?

Reply:
I was thinking the same thing, actually.

Reply:
Unfortunately, footers are plain text and don't render any Markdown.

![image](https://user-images.githubusercontent.com/37779854/151899593-b6ade2c8-7d1b-4da9-9254-9f8de1d7cd43.png)

Reply:
Oh, good to know.

> Unfortunately, footers are plain text and don't render any Markdown.
> 
> ![image](https://user-images.githubusercontent.com/37779854/151899593-b6ade2c8-7d1b-4da9-9254-9f8de1d7cd43.png)

Oh, good to know.

Reply:
Well then, perhaps you could still add a period, but ignore the backticks?

Reply:
I'm not opposed; done in https://github.com/python-discord/sir-lancebot/pull/1020/commits/851e6adb03e54244ef59c75a41783d9ec548ab69.